templates/dashboard/product/form.html

```diff
@@ -87,7 +87,9 @@
                   {{ product_form.price|materializecss }}
                 </div>
                 <div class="row">
-                  {{ product_form.tax_rate|materializecss }}
+                  {% if product_form.tax_rate.field.choices %}
```

Comment:
Do we want to hide this field?  This will change the structure of the form and can be misleading for users. We should show some popup with information that tax support is disabled and freeze the input.